core/markets/storage/client/client_deal.hpp

```diff
@@ -6,9 +6,14 @@
 #pragma once
 
 #include "vm/actor/builtin/types/market/deal.hpp"
+#include "markets/storage/mk_protocol.hpp"
 
 namespace fc::markets::storage::client {
   using vm::actor::builtin::types::market::ClientDealProposal;
+  using markets::storage::StorageDealStatus;
+  using markets::storage::DataRef;
```

Comment:
already in `fc::markets::storage` namespace
```suggestion
```

Reply:
no
I have compilation error without theese usings